packages/mux-video/src/index.ts

```diff
@@ -440,31 +457,18 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
           break;
         }
         /** In case newValue is an empty string or null, use this.autoplay which translates to booleans (WL) */
-        this.__core?.setAutoplay(this.autoplay);
+        this.#core?.setAutoplay(this.autoplay);
         break;
       case 'preload':
         if (newValue === oldValue) {
           break;
         }
-        this.__core?.setPreload(newValue as HTMLMediaElement['preload']);
+        this.#core?.setPreload(newValue as HTMLMediaElement['preload']);
         break;
       case Attributes.PLAYBACK_ID:
         /** @TODO Improv+Discuss - how should playback-id update wrt src attr changes (and vice versa) (CJP) */
         this.src = toMuxVideoURL(newValue ?? undefined, { domain: this.customDomain }) as string;
         break;
-      case Attributes.DEBUG: {
-        const debug = this.debug;
-        if (!!this.mux) {
-          /** @TODO Link to docs for a more detailed discussion (CJP) */
-          console.info(
```

Comment:
since isn't an issue anymore because setting this tears down hlsjs and mux-embed, right?

Reply:
indeed a `debug` change will reset playback core